admin_manual/configuration_ldap/ldap_proxy_cache_server_setup.rst

```diff
@@ -0,0 +1,110 @@
+=======================================================
+How To Install and Configure an LDAP Proxy-Cache Server
+=======================================================
+
+Background
+----------
+
+To reduce network traffic overhead and avoid problems either logging in or performing user searches while sharing, it's an excellent idea to implement an LDAP proxy cache.
+An LDAP proxy cache server, similar to other kinds of caching servers, is a special type of LDAP replica. It can cache a range of LDAP records, often resulting in improved LDAP server performance.
+
+Specifically, the records which need to be cached, for improved ownCloud performance, are:
+
+- Users that are allowed to log in
+- Groups (limited to the allowed users)
+- Quota
+- Mail
+- Search fields (e.g., ``sAMAccountName``, ``cn``, ``sn``, ``givenName``, and ``displayName``)
+
+How To Setup the Server
+-----------------------
+
+There's not that much to it, just the following five steps:
+
+1. Install OpenLDAP
+2. Configure the server
+3. Edit the Default Configuration Directory
+4. Perform a Test Search
+5. Check the Logs
+
```

Comment:
I would like to add a 6. Point here "ownCloud LDAP Configuration", to show and tell how to setup the ownCloud LDAP app